CHANGELOG_PENDING.md

```diff
@@ -21,5 +21,9 @@ Special thanks to external contributors on this release:
 ### IMPROVEMENTS:
 
 ### BUG FIXES:
-- [consensus] \#3297 Flush WAL on stop to prevent data corruption during
+
+* [consensus] \#3297 Flush WAL on stop to prevent data corruption during
   graceful shutdown
+* [cmd] [\#3314](https://github.com/tendermint/tendermint/issues/3314) Return an
```

Comment:
I think we run a script that adds the links in the end. Not sure if adding a link here directly does cause any problems with that script: https://github.com/tendermint/tendermint/blob/master/scripts/linkify_changelog.py

```suggestion
* [cmd] \#3314 Return an
```

Reply:
Gotcha. Amended. I saw a mix case usage in the changelog.